Sources/Core/Presenter/UploadingItem.swift

```diff
@@ -156,71 +135,44 @@ public final class UploaderItem: Equatable {
         extraData = nil
     }
     
-    private func createUploading() -> Observable<ProgressResponse<URL>> {
-        guard let channel = channel else {
-            return .empty()
-        }
-        
-        let request: Observable<ProgressResponse<URL>>
-        
+    private func encodeData() {
         if type == .file || type == .video {
             if let url = url, let data = try? Data(contentsOf: url) {
```

Comment:
The error is not handled correctly here. We shouldn't ignore the error from `Data(contentsOf:)` and use it on L144.